pkg/volume/empty_dir/empty_dir.go

```diff
@@ -168,26 +206,66 @@ func (ed *emptyDir) setupTmpfs(dir string) error {
 	// By default a tmpfs mount will receive a different SELinux context
 	// from that of the Kubelet root directory which is not readable from
 	// the SELinux context of a docker container.
-	//
-	// getTmpfsMountOptions gets the mount option to set the context of
-	// the tmpfs mount so that it can be read from the SELinux context of
-	// the container.
-	opts := ed.getTmpfsMountOptions()
-	glog.V(3).Infof("pod %v: mounting tmpfs for volume %v with opts %v", ed.podUID, ed.volName, opts)
+	var opts []string
+	if selinuxContext != "" {
```

Comment:
Can I just say how annoying it is to sprinkle selinux all over the codebase.  Is this REALLY the best we can do?